src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Refactoring/RenameEndpointTest.cs

```diff
@@ -337,22 +365,32 @@ public async Task Handle_Rename_MultipleFileUsages()
             Assert.Equal(2, textDocumentEdit2.Edits.Length);
         }
 
+        private async Task<DocumentContext> GetDocumentContextAsync(Uri file)
+        {
+            var documentContext = await _documentContextFactory.TryCreateAsync(file, CancellationToken.None);
```

Comment:
Why `await` a value just to return it here? Why not just remove `async` above and pass along the task?